pages/model/[uuid].tsx

```diff
@@ -554,6 +600,31 @@ function Model() {
           </Box>
         )}
       </Paper>
+      <Dialog open={ungovernedDialogOpen} onClose={handleUngovernedDialogClose}>
+        <DialogContent>
+          <Stack spacing={1}>
+            <Stack direction='row' spacing={2}>
+              <TextField
+                label='Deployment name'
+                variant='outlined'
+                value={ungovernedDeploymentName}
+                onChange={(event) => setUngovernedDeploymentName(event.target.value)}
+              />
+              <Button
+                disabled={ungovernedDeploymentName === ''}
+                variant='contained'
+                onClick={requestUngovernedDeployment}
+                autoFocus
+              >
+                Request
+              </Button>
+            </Stack>
+            <Typography sx={{ color: 'red' }} variant='caption'>
```

Comment:
Consider using `theme.error.main` instead of `red`

Reply:
For some reason even though theme seems to be correctly imported, it doesn't recognise "error". It definitely exists in the theme file, though...

Reply:
Ignore above!